app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -126,8 +126,11 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
     DisplayProperties.setShowVariants 0, showVariants
 
   $scope.addVariant = (product) ->
+    # Set new variant category to same as last product variant category to keep compactibility with deleted variant callback to set new variant category
+    newVariantId = $scope.nextVariantId();
+    newVariantCategoryId = product.variants[product.variants.length - 1]?.category_id
```

Comment:
I don't understand why we are doing this, It would make more sense to set the category to blank same as the new screen. I am not sure what you are referring to when you say "to keep compactibility with deleted variant callback to set new variant category"

Reply:
> I am not sure what you are referring to when you say "to keep compactibility with deleted variant callback to set new variant category"

I think he means to maintain the existing behaviour on the old screen. So there would be no apparent change when using the old screen. But as you say,

> It would make more sense to set the category to blank same as the new screen

Reply:
In the old screen the supplier is set to blank, I think it would be better to have a consistent behaviour.

Reply:
😮 Oh, so this change makes it worse than before? In that case we need to do better. 
Wandji would it be hard to make the old screen behave like the new one?

Reply:
Thanks a lot, @dacook and @rioug for identifying this one.
I noticed that the columns preference default for the category column in old screens have `visible: false`
Not so sure why we have this by default but I will gladly appreciate your input on this again.

Reply:
Ok, that's a good point Wandji. If the Category column is hidden by default, it's not intuitive for the user to choose it.
I just tried it out and can see what you mean. 

If your PR ensures the old screen maintains the existing behaviour, I think it's ok to proceed as it is.  It adds a bit of technical debt, but that's ok because we're planning to remove the old screen later, once the transition to the new screen is complete.

Is that ok with you Gaetan?

Reply:
> Is that ok with you Gaetan?

Yeah that's fine, if we get complaint we can always tell them to use the new screen :wink: